client/i18n/locales/english/intro.json

```diff
@@ -1945,10 +1956,18 @@
       },
       "wfyg": { "title": "144", "intro": [] },
       "guqy": { "title": "145", "intro": [] },
-      "nkge": { "title": "146", "intro": [] },
+      "quiz-javascript-math": {
```

Comment:
I wonder if this should be math object?

Reply:
I was going to do that, but looking at the sheet - I believe it was more than that - for instance, basic operations, and order of operations, etc. (not Math object) - but it does also include some of the Math object. So I just went with `Math`. What do you think?

Reply:
yeah, we can just leave it to math then 👍